### PR TITLE
Add GitHub issue template for user reported bugs

### PR DESCRIPTION
**Pull request by wiz**

### CODE REVIEW

**Comment by wiz** [comment]
Preview at https://github.com/wiz/bisq/issues/new

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+### Description
+
+<!-- brief description of the bug -->
+
+#### Version
+
+<!-- commid id or version number -->
```
Comment (ripcurlx) [inline .github/ISSUE_TEMPLATE.md]: --> commit

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - just a small typo. Besides that utACK

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (wiz) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
<!-- commit id or version number -->
```

**Review by ripcurlx — APPROVED** [review]

**Pull request merged**